tests/Unit/Core/ConstraintValidator/CleanHtmlValidatorTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Unit\Core\ConstraintValidator;
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\CleanHtmlValidator;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\CleanHtml;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class CleanHtmlValidatorTest extends ConstraintValidatorTestCase
+{
+    /**
+     * @dataProvider getValuesWithScriptTags
```

Comment:
If it used only once and is not too big, please avoid using `@dataProvider`. This is because
- It makes the test a little harder to read as you have to back-and-forth between the test and the dataProvider to understand the whole. Unit tests can be hard to read so it's good to keep the logic as a whole statement inside the test, like a scenario.
- We have detected performance issues with dataProviders, due to some inner behavior of phpunit 😞